src/main/java/SearchEngine.java

```diff
@@ -0,0 +1,109 @@
+import impl.AllStrategyImpl;
+import impl.AnyStrategyImpl;
+import impl.NoneStrategyImpl;
+import strategies.SearchStrategy;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.stream.Stream;
+
+public class SearchEngine {
```

Comment:
Can you use the default access modifier?

Reply:
Please add an empty line after the class definition.

Reply:
fixed